config/default/webhookcainjection_patch.yaml

```diff
@@ -0,0 +1,15 @@
+# This patch add annotation to admission webhook config
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  labels:
+    app.kubernetes.io/name: mutatingwebhookconfiguration
```

Comment:
Those labels can be removed as there are added using common labels.

Reply:
These were generated, so I was unsure about removing them